go.mod

```diff
@@ -37,8 +37,8 @@ require (
 	golang.org/x/oauth2 v0.4.0
 	golang.org/x/sync v0.1.0
 	gomodules.xyz/jsonpatch/v2 v2.2.0
-	google.golang.org/api v0.97.0
-	helm.sh/helm/v3 v3.10.3
+	google.golang.org/api v0.103.0
```

Comment:
note: it's unavoidable to pull in some google dep updates with the helm version bump. I think it's worthwhile to do it now anyway, assuming all tests are passing.

There are several other dependency upgrades which are hard to avoid here!